### PR TITLE
CIV-10782 Bulk print hearing

### PR DESCRIPTION
**Pull request by miguelMolina3691**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-10782

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by maciejmodzelewski — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java [hunk 1]
```diff
@@ -292,7 +292,8 @@ public enum CaseEvent {
     SEND_CVP_JOIN_LINK(USER),
     SET_LIP_RESPONDENT_RESPONSE_DEADLINE(CAMUNDA),
     NOTIFY_LIP_RESPONDENT_CLAIMANT_CONFIRM_TO_PROCEED(CAMUNDA),
-    NOTIFY_LIP_APPLICANT_CLAIMANT_CONFIRM_TO_PROCEED(CAMUNDA);
+    NOTIFY_LIP_APPLICANT_CLAIMANT_CONFIRM_TO_PROCEED(CAMUNDA),
+    SEND_HEARING_TO_LIP_DEFENDANT(CAMUNDA);
```
Comment (m-meulendijks-v1) [inline src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java:296]: Do we also need events for hearing to Claimant LIP + Final orders for claimant & defendant?

Comment (miguelMolina3691) [reply src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java:296]: I will confirm if it is included in these developments

Comment (miguelMolina3691) [reply src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java]: Included in code

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by maciejmodzelewski — DISMISSED** [review]

**Comment by HarryH96** [comment]
Dev Blocked

**Review by m-meulendijks-v1 — APPROVED** [review]

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**